src/server/server/templates/base.html

```diff
@@ -2,12 +2,12 @@
 {% autoescape true %}
 <html>
     <head>
-        <link type="text/css" rel="stylesheet" href="/stylesheets/main.css" />
+        <link type="text/css" rel="stylesheet" href="/static/main.css" />
```

Comment:
Do you mean instead or under "static" folder ?
"static" is the standard structure for flask framework
If you mean a nested structure under static, it seemed to me a little overkill for 2 files, but it's fine to me.